Patch based on the upstream commit described below, adjusted to apply to the
Debian package by Peter Michael Green.

commit 58ba720df859b8d8a4c685a7984266e6f237b75e
Author: Yuki Okushi <jtitor@2k36.org>
Date:   Wed May 12 11:47:06 2021 +0900

    Remove unnecessary the `const_fn` feature gate

diff --git a/src/lib.rs b/src/lib.rs
index 1508086ff..2baeef375 100644
--- a/src/lib.rs
+++ b/src/lib.rs
@@ -216,6 +216,5 @@
 #![feature(
     repr_simd,
-    const_fn,
     platform_intrinsics,
     stdsimd,
     aarch64_target_feature,
